python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -0,0 +1,442 @@
+from typing import List
+import binascii
+
+import numpy
+import pandas
+import sqlalchemy as sa
+
+from lamp_py.postgres.postgres_utils import DatabaseManager
+from lamp_py.postgres.postgres_schema import (
+    VehicleTrips,
+    TempHashCompare,
+)
+from lamp_py.runtime_utils.process_logger import ProcessLogger
+from .l1_cte_statements import (
+    get_rt_trips_cte,
+    get_static_trips_cte,
+)
+
+
+def insert_trips_hash_columns(
+    db_manager: DatabaseManager, events: pandas.DataFrame
+) -> None:
+    """
+    Load new 'trip_hash' records and columns into RDS
+
+    This guarantees that all events represented in the events dataframe will have
+    matching trips data in the VehicleTrips table
+    """
+    insert_columns = [
+        "trip_hash",
+        "fk_static_timestamp",
+        "direction_id",
+        "route_id",
+        "start_date",
+        "start_time",
+        "vehicle_id",
+    ]
```

Comment:
This function will insert basic trips information for any new trips no currently represented in the RDS.